src/resources/js/dependency.js

```diff
@@ -72,6 +75,7 @@
 						}
 					} else {
 						$dependent
+							.hide()
```

Comment:
Yeah, I know, the thing is, if we go with hiding it with JS, we will lose a lot of flexibility that the class gives us.

I mean, I would say that we could include that new `.wrapper .tribe-dependent` on the `.pcss`.